profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single Profile
+**GET /api/cris/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the profile id that corresponds to the id of the eperson associated
+* visible: indicates whether the profile is public (visible: true) or private (otherwise)
+* type: the resource type (profile)
+
+Exposed links:
+* item: the item with which the profile is modeled
```

Comment:
We may want to explicitly note here that this Item **must be a Person Entity**.  So, you should not be able to link a normal Item or a Publication/Project Entity as a "profile".

Reply:
@LucaGiamminonni : I'm ok with leaving this more generic, provided it's clear how to configure the EntityType which corresponds to a Profile.  However, I think it should default to `Person` in DSpace, because that's the most logical setting for the Profile type.